tools/templates/docstring.jinja2

```diff
@@ -30,7 +30,7 @@ Parameters
     {% if (attr.attr_constructor == 'AttrGraph') and (attr.subgraph_solution is none) %}
 {{ attr.name }}_input_types: Iterable[Type]
     Specifies the input types for the subgraph ``{{ attr.name }}``.
-    Non-standard parameter created by the opset generator, as inference (a solution) it was not implemented or is impossible.
+    Non-standard parameter created by the opset generator, as constructing a graph without this information is not implemented or impossible.
```

Comment:
Why are we patching this? I agree it's a bit incomprehensible the way it's written upstream, but I think that should be resolved in a PR to the ONNX repository.